scss/bootstrap/variants/_outlined-buttons.scss

```diff
@@ -1,11 +1,7 @@
-@import 'bootstrap/mixins/button-outline-variant';
-@import 'bootstrap/mixins/hover';
-@import 'bootstrap/overrides/variables';
+@import '../overrides/variables';
 
 .btn-default-outline {
   @include button-outline-variant($crm-white);
-  border-color: $crm-white !important;
-  color: $crm-white !important;
```

Comment:
It was there by a reason I think. You would need to run BackstopJS. If you are not sure you have time for that I would recommend to revert that change.

Reply:
@igorpavlov this variation was introduced in https://github.com/compucorp/org.civicrm.shoreditch/pull/21, specifically for the L&A dashboard page https://github.com/compucorp/civihr/pull/1976 . In the end that page is now using a different button (`.btn-lg.btn-primary`) and nowhere else is the button used.

In any case just to be sure I checked the changes both on the styleguide and also on the L&A dashboard page (by manually changing the class on the button from `.btn-primary` to `.btn-default-outline`) and I don't see any difference in any of the different states (hover, focus, active, etc)

Reply:
Please show in the PR that the changes were tested manually.